.golangci.yml

```diff
@@ -86,6 +85,10 @@ linters-settings:
       - "encoding/*"
       - "gopkg.in/yaml.v2*"
 
+  cyclop:
+    max-complexity: 15
+    skip-tests: true
```

Comment:
did we have issues with this in tests? I'll be in favor of keeping it.

Reply:
Sure, I don't mind, was mostly out of concern for making stuff faster

Reply:
:thinking: keeping branching low in tests should be a requirement though you make a point. We could try to track which linter takes time, could be interesting.